File: w4/php/multipleN.php
<?php
    $multiple = intval (fgets(STDIN));
    $divisor = intval (fgets(STDIN));
    $out = '';
    
    $multiple -= $multiple % $divisor;
    
    for ( $i = 0; $i < $multiple; $i += $divisor ) {
        $out .= $i . ' ';
    }
    $out .= $multiple;
    echo $out . PHP_EOL;
?>

File: w1/multipleN.c
#include <stdio.h>

int main() {
    int multiple, divisor;
    int i = 0;

    scanf("%d %d", &multiple, &divisor);
    if (divisor < 0)
    {
        divisor = -divisor;
    }
    for ( ; i <= (multiple - divisor); i += divisor ) {
        printf("%d ", i);
    }
    printf("%d\n", i);
    return 0;
}
